config/datanode.example.toml

```diff
@@ -41,29 +41,26 @@ tcp_nodelay = true
 # WAL data directory
 provider = "raft_engine"
 
-# Raft-engine wal options, see `standalone.example.toml`
+# Raft-engine wal options, see `standalone.example.toml`.
 # dir = "/tmp/greptimedb/wal"
 file_size = "256MB"
 purge_threshold = "4GB"
 purge_interval = "10m"
 read_batch_size = 128
 sync_write = false
 
-# Kafka wal options.
-# The broker endpoints of the Kafka cluster. ["127.0.0.1:9090"] by default.
+# Kafka wal options, see `standalone.example.toml`.
 # broker_endpoints = ["127.0.0.1:9090"]
-# Number of topics shall be created beforehand.
 # num_topics = 64
-# Topic name prefix.
-# topic_name_prefix = "greptimedb_wal_kafka_topic"
-# Number of partitions per topic.
+# topic_name_prefix = "greptimedb_wal_topic"
```

Comment:
Looks like the datanode doesn't need this parameter.

Reply:
Removed unnecessary wal configs of datanode.